.github/workflows/twister_tests.yml

```diff
@@ -9,7 +9,7 @@ on:
     - main
     - v*-branch
     paths:
-    - 'scripts/pylib/twister/**'
+    - 'scripts/pylib/**'
```

Comment:
is it safe to take everything from `scripts/pylib` instead of explicitly choose like this:

```suggestion
    - 'scripts/pylib/twister/**'
    - 'scripts/pylib/build_helpers/**'
```

Reply:
You're right, changed that to explicit `build_helpers` mention.